recipes/cutensor/meta.yaml

```diff
@@ -0,0 +1,97 @@
+  url: https://github.com/JuliaBinaryWrappers/CUTENSOR_CUDA102_jll.jl/releases/download/CUTENSOR_CUDA102-v{{ version }}%2B0/CUTENSOR_CUDA102.v{{ version }}.x86_64-w64-mingw32.tar.gz  # [win64 and cuda_compiler_version == "10.2"]
+  url: https://github.com/JuliaBinaryWrappers/CUTENSOR_CUDA110_jll.jl/releases/download/CUTENSOR_CUDA110-v{{ version }}%2B0/CUTENSOR_CUDA110.v{{ version }}.x86_64-w64-mingw32.tar.gz  # [win64 and cuda_compiler_version == "11.0"]
+  sha256: 5efce496c330cd9f6df38b93e605127977f0221ca023a4c864bb9e2bc48dd595  # [win64 and cuda_compiler_version == "10.1"]
+  sha256: 23830ae8da7eaaaef63b572ff19a47f3a86ca7124b8ce2c85bd76286cbf58014  # [win64 and cuda_compiler_version == "10.2"]
+  sha256: a2ff02ca1003f6e96e63335e6296637173d02cf9d44892225815679dc9c13ee7  # [win64 and cuda_compiler_version == "11.0"]
+
+build:
+  number: 0
+  skip: True  # [cuda_compiler_version in ("None", "9.2", "10.0")]
+  script:
+    - mkdir -p $PREFIX/include                                            # [unix]
+    - cp -r include/* $PREFIX/include/                                    # [unix]
+    - mkdir -p $PREFIX/lib                                                # [unix]
+    - mv lib/libcutensor.so* $PREFIX/lib/                                 # [unix]
+
+    - mkdir %LIBRARY_INC%                                                 # [win64]
+    - copy %SRC_DIR%\\include\\cutensor.h %LIBRARY_INC%\\                 # [win64]
+    - mkdir %LIBRARY_INC\\cutensor%                                       # [win64]
+    - copy %SRC_DIR%\\include\\cutensor\\types.h %LIBRARY_INC%\\cutensor  # [win64]
+    - mkdir %LIBRARY_BIN%                                                 # [win64]
+    - copy %SRC_DIR%\\bin\\cutensor.dll %LIBRARY_BIN%\\                   # [win64]
+  ignore_run_exports_from:
+    - {{ compiler('c') }}    # [linux]
+  run_exports:
+    - {{ pin_subpackage('cutensor') }}
+  missing_dso_whitelist:
+    - '*/libgomp.*'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cuda') }}
+    - sysroot_linux-64 2.17  # [linux64]
+    - libgomp  # [linux]
+  run:
+    # Only GCC_3.0 or older symbols present
+    - libgcc-ng >=3.0     # [linux]
+    # Only GLIBCXX_3.4 or older symbols present
+    - libstdcxx-ng >=3.4  # [linux]
+  run_constrained:
+    # Only GLIBC_2.17 or older symbols present
+    - __glibc >=2.17      # [linux]
+
+test:
+  commands:
+    - test -f $PREFIX/include/cutensor.h                    # [linux]
+    - test -f $PREFIX/include/cutensor/types.h              # [linux]
+    - test -f $PREFIX/lib/libcutensor.so                    # [linux]
+
+    - if not exist %LIBRARY_INC%\\cutensor.h exit 1         # [win]
+    - if not exist %LIBRARY_INC%\\cutensor\\types.h exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\\cutensor.dll exit 1       # [win]
```

Comment:
Check for import library here

Reply:
You meant check for stuff in `LIBRARY_LIB`? But nothing is copied to there?

Reply:
If it's not there, you can't link to it. You'll have to generate it using the librarian tool. See https://stackoverflow.com/a/9946390/4768820

Reply:
Oh...Interesting. I know nothing about Windows. Let me give it a shot!

Reply:
I want to take a stupid approach: Upload the lib files as patch (each is only 8KB).

Reply:
Done in d89385c.

Reply:
Please don't. We'll be distributing proprietary files in this git repo without packaging the license file. Also please delete the commit instead of reverting if you want this git branch to be merged.

Reply:
I thought we set `license_file` already? What if I also upload the license file? If not, then I'll force push to clean up, and then comment out the Windows branch.

Reply:
we'll be distributing the `.lib` file in this git repo's history. even if you upload the license file, we'll still be distributing the `.lib` standalone without the license file.

Reply:
I assume you're saying if I upload `.lib`s, the license file must also be uploaded in the same commit. I can do that too, but since I'm gonna force push, I may as well just clean it up.

Reply:
No, that's not okay. the `.lib` file can still be downloaded with a direct link without the license file.

Reply:
What's not ok? Force push?

Reply:
Having the `.lib` file in the git repository.

Reply:
OK I force pushed. Sorry for the mess.